cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -893,6 +893,14 @@
     </description>
   </property>
 
+  <property>
+    <name>data.storage.sql.scan.size.rows</name>
+    <value>100</value>
+    <description>
+      The number of rows fetched for database reads from Postgres sql.
```

Comment:
Can you change Postgres sql to PostgreSQL?

Reply:
Changed